lib/graph3d/Graph3d.js

```diff
@@ -1680,6 +1646,40 @@ Graph3d.prototype._getColorsSize = function() {
 };
 
 
+/**
+ * Determine the color corresponding to a given value on the color scale.
+ *
+ * @param {number} [x] the data value to be mapped running from 0 to 1
+ * @param {number} [v] scale factor between 0 and 1 for the color brightness
+ * @returns {string}
+ * @private
+ */
+Graph3d.prototype._colormap = function(x, v) {
+  if (v === undefined) {
+    v = 1;
+  }
```

Comment:
Could you use a bit more modern ES, please? This could be done as `function(x, v = 1) {`.

Reply:
Ok, thanks for the tip!